examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -57,6 +57,7 @@ const connectionProps: AzureConnectionConfig = useAzure ? {
 // Define the schema of our Container.
 // This includes the DataObjects we support and any initial DataObjects we want created
 // when the container is first created.
+
 const containerSchema = {
```

Comment:
i think we should find a way to leverage this. i don't actually like that i need to pass this to both create and load. i feel like we should input this in a single place earlier in the pipeline, and then infer the type from here